compiler/modules/front.syntax/ast.bal

```diff
@@ -476,9 +476,8 @@ public type BinaryTypeOp "|" | "&";
 public type BinaryTypeDesc record {|
     *PositionFields;
     BinaryTypeOp op;
-    Position opPos;
-    TypeDesc left;
-    TypeDesc right;
+    Position[] opPositions;
```

Comment:
I think `opPos` is least bad choice here.

Reply:
fixed